DiffPlex/Chunkers/LineEndingsPreservingChunker.cs

```diff
@@ -4,7 +4,11 @@ namespace DiffPlex.Chunkers
 {
     public class LineEndingsPreservingChunker:IChunker
     {
-        private readonly string[] emptyArray = new string[0];
+#if NETSTANDARD1_3_OR_GREATER || NET46_OR_GREATER || NETCOREAPP
```

Comment:
Array.Empty is better than creating a new array, but since this library still targets net35, net40, and netstandard1.0 it's not always available. I know the change isn't really needed though since this code isn't performance-critical.

Reply:
Lets remove for now then